[2025-07-23T00:27:44.405+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-22T00:00:00+00:00 [queued]>
[2025-07-23T00:27:44.414+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-22T00:00:00+00:00 [queued]>
[2025-07-23T00:27:44.415+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-23T00:27:44.425+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres> on 2025-07-22 00:00:00+00:00
[2025-07-23T00:27:44.432+0000] {standard_task_runner.py:60} INFO - Started process 18751 to run task
[2025-07-23T00:27:44.435+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sales_batch_etl', 'load_to_postgres', 'scheduled__2025-07-22T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/sales_batch_etl.py', '--cfg-path', '/tmp/tmp61oryo2v']
[2025-07-23T00:27:44.437+0000] {standard_task_runner.py:88} INFO - Job 28: Subtask load_to_postgres
[2025-07-23T00:27:44.453+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-23T00:27:44.514+0000] {task_command.py:423} INFO - Running <TaskInstance: sales_batch_etl.load_to_postgres scheduled__2025-07-22T00:00:00+00:00 [running]> on host e17dc2585651
[2025-07-23T00:27:44.778+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_batch_etl' AIRFLOW_CTX_TASK_ID='load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-07-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-22T00:00:00+00:00'
[2025-07-23T00:27:44.780+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-23T00:27:44.783+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/load_to_postgres.py']
[2025-07-23T00:27:44.812+0000] {subprocess.py:86} INFO - Output:
[2025-07-23T00:27:46.662+0000] {subprocess.py:93} INFO - üì§ Loading data from /tmp/sales.csv to PostgreSQL...
[2025-07-23T00:27:46.669+0000] {subprocess.py:93} INFO - üìä Read DataFrame with shape: (10000, 6)
[2025-07-23T00:27:46.670+0000] {subprocess.py:93} INFO - üõ†Ô∏è Creating table if not exists...
[2025-07-23T00:27:46.672+0000] {subprocess.py:93} INFO - üì• Inserting rows into sales_staging table...
[2025-07-23T00:27:46.673+0000] {subprocess.py:93} INFO - ‚úÖ Data inserted successfully into PostgreSQL.
[2025-07-23T00:27:46.826+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-23T00:27:46.892+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sales_batch_etl, task_id=load_to_postgres, execution_date=20250722T000000, start_date=20250723T002744, end_date=20250723T002746
[2025-07-23T00:27:46.944+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-23T00:27:46.982+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
